tests/v2/validation/rbac/globalrolesv2/globalroles_v2_test.go

```diff
@@ -0,0 +1,484 @@
+	require.NoError(gr.T(), err)
+
+	grbName, expectedClusterCount := gr.testValidateRBACResources(createdUser, inheritedClusterRoles)
+
+	log.Info("Verify that the user can list all the downstream clusters.")
+	userClient, err := gr.client.AsUser(createdUser)
+	require.NoError(gr.T(), err)
+	clusterList, err := userClient.Steve.SteveType(clusters.ProvisioningSteveResourceType).ListAll(nil)
+	require.NoError(gr.T(), err)
+	actualClusterCount := len(clusterList.Data)
+	require.Equal(gr.T(), expectedClusterCount, actualClusterCount, "Unexpected number of Clusters: Expected %d, Actual %d", expectedClusterCount, actualClusterCount)
+
+	log.Info("Delete Global Role Binding.")
+	err = rbac.DeleteGlobalRoleBinding(gr.client, grbName)
+	require.NoError(gr.T(), err)
+
+	log.Info("Verify that the global role is not deleted.")
+	listOpt := metav1.ListOptions{
+		FieldSelector: "metadata.name=" + createdGlobalRole.Name,
+	}
+	grList, err := rbac.ListGlobalRoles(gr.client, listOpt)
+	require.NoError(gr.T(), err)
+	require.NotEmpty(gr.T(), grList, "Global Role does not exist.")
+
+	log.Info("Verify that the global role binding is deleted for the user.")
+	grbOwner, err := getGlobalRoleBindingForUser(gr.client, createdUser.ID)
+	require.NoError(gr.T(), err)
+	require.Empty(gr.T(), grbOwner, "Global Role Binding exists for the user.")
+
+	log.Info("Verify that the cluster role template bindings are deleted for the downstream clusters.")
+	expectedCrtbCount := 0
+	crtbs, err := listClusterRoleTemplateBindingsForInheritedClusterRoles(gr.client, grbOwner, expectedCrtbCount)
+	require.NoError(gr.T(), err)
+	actualCrtbCount := len(crtbs.Items)
+	require.Equal(gr.T(), expectedCrtbCount, actualCrtbCount, "Unexpected number of ClusterRoleTemplateBindings: Expected %d, Actual %d", expectedCrtbCount, actualCrtbCount)
+
+	log.Info("Verify that the cluster role bindings are deleted for the downstream cluster.")
+	crbs, err := getCRBsForCRTBs(gr.client, crtbs)
+	require.NoError(gr.T(), err)
+	actualCrbCount := len(crbs.Items)
+	require.Equal(gr.T(), 0, actualCrbCount, "Unexpected number of ClusterRoleBindings: Expected %d, Actual %d", 0, actualCrbCount)
+
+	log.Info("Verify that the role bindings are deleted for the downstream cluster.")
+	rbs, err := getRBsForCRTBs(gr.client, crtbs)
+	require.NoError(gr.T(), err)
+	actualRbCount := len(rbs.Items)
+	require.Equal(gr.T(), 0, actualRbCount, "Unexpected number of RoleBindings: Expected %d, Actual %d", 0, actualRbCount)
+
+	log.Info("Verify that the user cannot list the downstream clusters.")
+	clusterList, err = userClient.Steve.SteveType(clusters.ProvisioningSteveResourceType).ListAll(nil)
+	require.NoError(gr.T(), err)
+	actualClusterCount = len(clusterList.Data)
+	require.Equal(gr.T(), 0, actualClusterCount, "Unexpected number of Clusters: Expected %d, Actual %d", 0, actualClusterCount)
+}
+
+func (gr *GlobalRolesV2TestSuite) TestDeleteInheritedClusterRoles() {
```

Comment:
since you're creating users with iCRs can you update the test name to reflect that this creates and deletes resources/users?